src/store/modules/Overview/OverviewStore.js

```diff
@@ -0,0 +1,33 @@
+import api from '../../api';
+
+const OverviewStore = {
+  namespaced: true,
+  state: {
+    serverInfo: '--'
```

Comment:
Following up with my comment on Overview.vue component how do you feel about separating each serverInfo property into its own state property?
```
state: {
  serverModel: '--',
  serverManufacturer: '--',
  serverSerialNumber: '--'
}
```
And can make use of mapState in the Overview.vue component